package.json

```diff
@@ -49,8 +49,8 @@
     "lint:scripts": "cd scripts && substrate-dev-run-lint"
   },
   "dependencies": {
-    "@polkadot/api": "^5.7.1",
-    "@polkadot/apps-config": "^0.95.2-50",
+    "@polkadot/api": "^5.8.3",
```

Comment:
Only grumble is the resolutions need to match the api version, so you can bump the api version to 5.8.4.-1 to make it easiest